app/models/offspring.rb

```diff
@@ -3,4 +3,5 @@ class Offspring < ApplicationRecord
   validates :first_name, presence: true, length: { within: 2..60 }
   validates :last_name, presence: true, length: { within: 2..60 }
   validates :user, presence: true
+  has_many :assignments, dependent: :destroy
```

Comment:
No, I don't think so, I would prefer a history table (but not know)

1. You get a version number in the assignment
2. You store in a table the history of the assignment

So, change it to has_one and we will do the historic in the future